food-truck-api/src/main/java/food/truck/api/recommendation/StrategySelector.java

```diff
@@ -17,7 +17,14 @@ public StrategySelector(TruckService truckSvc, SubscriptionService subSvc) {
 
     public TruckRecommendationStrategy selectStrategy(AbstractUser u, UserPreferences prefs) {
         if (u instanceof User) {
+            if(!prefs.isActive()){
+               return new PassiveUserRecommendationStrategy(truckSvc, subSvc, (User) u, prefs);
+            }
             return new ScoringRecommendationStrategy(truckSvc, subSvc, (User) u, prefs);
+
+        }
+        if(!prefs.isActive()){
+            return new PassiveGuestRecommendationStrategy(truckSvc, u.getPosition(), prefs);
         }
         return new GuestRecommendationStrategy(truckSvc, u.getPosition());
```

Comment:
The `GuestRecommendationStrategy` was already a passive strategy, as the user could not input desired tags, price, etc. and have it account for these in selection. So we currently don't have an active guest strategy.